src/components/LoginPage.js

```diff
@@ -219,7 +222,8 @@ const LoginPage = ({ theme }) => {
             name="base_url"
             component={renderInput}
             label={translate("synapseadmin.auth.base_url")}
-            disabled={loading}
+            disabled={override_server ? true : loading}
```

Comment:
IMHO this is better readable:
```suggestion
            disabled={override_server || loading}
```